src/components/Footer/__snapshots__/Footer.test.tsx.snap

```diff
@@ -273,7 +273,7 @@ exports[`Footer renders the UI snapshot correctly 1`] = `
     >
       <p>
         © The New York Public Library, 
-        2024
+        2025
```

Comment:
What is setting this value in the app?

Reply:
It's a function that sets it to the current year. So it does it automatically.